cursorless-talon/src/csv_overrides.py

```diff
@@ -126,35 +124,44 @@ def on_watch(path, flags):
             default_values,
             super_default_values,
             extra_ignored_values,
-            allow_unknown_values,
             default_list_name,
             pluralize_lists,
             ctx,
         )
 
-    def unsubscribe():
+    def unsubscribe() -> None:
         fs.unwatch(str(file_path.parent), on_watch)
 
     return unsubscribe
 
 
-def check_for_duplicates(filename, default_values):
+@dataclass
+class ListValue:
+    key: str
+    value: str
+    #: The list name.
+    list: str
+
+
+def check_for_duplicates(
+    filename: str, default_values: Mapping[str, Mapping[str, str]]
+):
     results_map = {}
```

Comment:
I can't see what type this is supposed to be... because I don't think any items are ever assigned to this :confused:

I recall this bit was in one of the other functions previously. Think this ended up being broken after a recent refactor.